_posts/2020-08-10-changelog-128.markdown

```diff
@@ -0,0 +1,177 @@
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5736 -->
+    * Search and update changed references when moving a top-level item.
+    Previously, [`Search for references`](https://www.jetbrains.com/help/idea/move-file-dialog.html) checkbox did not work
+    and all the references remained the same after refactoring
+        
+        {% include gif-img.html path="/assets/posts/changelog-128/move-update-references" w="600px" %}
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5484 -->
+    * Take re-exports into account when moving a single file
+
+        {% include gif-img.html path="/assets/posts/changelog-128/move-reexports" w="600px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5758 -->
+* Provide `true`/`false` completion for boolean values (by [@ozkriff])
+
+    {% include gif-img.html path="/assets/posts/changelog-128/bool-completion" w="400px" %}
+
+* Improve `Wrong number of type arguments` inspection
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5811 -->
+    * Support `dyn Trait` and `impl Trait` (by [@Kobzol])
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5306 -->
+    * Provide `Add missing type arguments` fix (by [@Kobzol]).
+    Currently, the fix adds only type parameters and ignores lifetimes and associated types
+
+        {% include gif-img.html path="/assets/posts/changelog-128/missing-type-arg" w="400px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5643 -->
+* Add `Substitute associated type` intention (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-128/subst-assoc-type" w="700px" %}
+
+* Improve struct patterns completion:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5660 -->
+    * Suggest only struct fields while completion in struct patterns (by [@Kobzol])
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5825 -->
+    * Completion for `..` syntax in struct patterns (by [@Kobzol])
+
+* Type inference improvements:    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5808 -->
+    * Infer types for [`associated_type_bounds`](https://github.com/rust-lang/rfcs/blob/master/text/2289-associated-type-bounds.md) rustc nightly feature
+    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5733 -->
+    * Infer type for types implementing the `Try` trait when using question mark operator (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5798 -->
+* Add `Main function not found in crate` ([E0601](https://doc.rust-lang.org/error-index.html#E0601)) inspection for build scripts
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5747 -->
+* Fold external (non-project related) parts of backtraces (by [@Kobzol])
+
+## Performance Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5790 -->
+* Avoid resolve in `RsVisible.isPublic` for restricted visibility
```

Comment:
```suggstion
* Avoid name resolution while checking if an item is public or not. It should slightly improve performance of plugin analysis
```

What do you think?

Reply:
I like it, probably except `plugin analysis`: maybe `code insight`?

Reply:
It's ok for me as well